index.js

```diff
@@ -1,21 +1,15 @@
 var loaderUtils = require('loader-utils');
 var MessageFormat = require('messageformat');
 
-module.exports = function(content) {
-  var query = loaderUtils.parseQuery(this.query);
-  var locale = query.locale || 'en';
-  var messages = (Array.isArray(this.inputValue) && typeof this.inputValue[0] === 'object') ? this.inputValue[0] : this.exec(content, this.resource);
```

Comment:
I thought some more about it, and I really think the right approach is to have this only consume pure JSON and fix `multi-json-loader` to output that instead of JS. I'll merge this.